assets/styles/main.scss

```diff
@@ -311,3 +311,5 @@ figure {
 .u-space--top--zero {
   margin-top: 0 !important;
 }
+
+@import "./hero-carousel";
```

Comment:
This part is already in the `v3` branch and shouldn't be  part of this pull request.